inviteme/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/gorilla/schema"
+	"net/http"
+	"net/url"
+)
+
+const (
+	inviteLink    string = "https://jdsb-test-ground-slack-invite.herokuapp.com/"
+	imageLink     string = "http://giphygifs.s3.amazonaws.com/media/5q7XU3MurT0pW/giphy.gif"
+	imageText     string = "I came in like a wrecking ball"
+	imageAltText  string = "Picture of a cat wrecking neatly stacked books"
+	linkMsg       string = "Click on the link below to get an invite to *jdsb-wrecking-ball* workspace: "
+	myJdsbSlackID        = "UGAA0BJ4R"
+)
+
+var (
+	decoder = schema.NewDecoder()
+)
+
+func handler(request events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	parsedBody, err := url.ParseQuery(request.Body)
+	if err != nil {
+		return &events.APIGatewayProxyResponse{
```

Comment:
I just caught that I am not using my actual error messages.

When encountering errors, I should probably be returning a nil Response, and an error that looks something like the following?:

```
errMsg := fmt.Sprintf("[ERROR] %v", err)
return nil, errors.New(errMsg)
```